images/ansible-operator/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN yum clean all && rm -rf /var/cache/yum/* \
   && yum -y update \
   && yum install -y libffi-devel openssl-devel python38-devel gcc python38-pip python38-setuptools \
   && pip3 install --no-cache-dir \
+    cryptography==3.3.1 \
```

Comment:
Have we the fragment over this change? I could not find it? 
So, wdyt about to do a follow-up pr with that?

Reply:
This change isn’t user-facing, see https://github.com/operator-framework/operator-sdk/pull/4510#issue-571546049.